packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -494,7 +509,7 @@ export enum TelemetryDataTag {
  */
 export function isTaggedTelemetryPropertyValue(x: any): x is ITaggedTelemetryPropertyType {
     return (typeof(x?.value) !== "object" && typeof(x?.tag) === "string");
-}
+} // Q: could the first check instead be against taggedtelemetrytype itself?
```

Comment:
Not sure what you mean... feel free to propose a change here

Reply:
...rereading, dumb question